spec/support/acceptance/supported_versions.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 def supported_versions
-  %w[5.0 6.0]
+  # this is a hack so that we don't have to rewrite the other tests
+  [ENV.fetch('BEAKER_FACTER_ZABBIX_VERSION')]
```

Comment:
It uses the env var verbatim, so it's 
```suggestion
  [ENV.fetch('BEAKER_FACTER_zabbix_version')]
```

We can debate whether it should make it case insensitive and always uppercase the env var while lower casing the fact, but it's not here today.

Reply:
just realized the same when running m2gha locally.